.bldr.toml

```diff
@@ -756,21 +756,38 @@ plan_path = "ponysay"
 plan_path = "popt"
 [postgresql]
 plan_path = "postgresql"
+[postgresql-client]
+plan_path = "postgresql"
 [postgresql93]
 plan_path = "postgresql93"
 paths = [
   "postgresql/*",
 ]
+[postgresql93-client]
+plan_path = "postgresql93-client"
+paths = [
+  "postgresql/*",
```

Comment:
I think this should be `postgresql93/*` and `postgresql93-client/*`,  right?   so that way a rebuild is triggered on any changes to the 9.3 server or client path.  maybe we also want to trigger a rebuild from `postgresql/*` although I'm not entirely sure if i see the value in that.